### PR TITLE
Add multi branch support 

### PR DESCRIPTION
**Pull request by Alea81**

### CODE REVIEW

**Comment by Alea81** [comment]
Will be rebased after #32

File: src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs [hunk 1]
```diff
@@ -80,6 +80,38 @@ public static bool DoesSubTypeMatch(this PolicyConfiguration configuration, Poli
...
 
+        /// <summary>
+        /// Flattens the policies by branches.
+        /// </summary>
+        /// <param name="policies">Policies</param>
+        /// <param name="serializer">Json serializer</param>
+        /// <returns>Flattened policies</returns>
+        public static IEnumerable<Policy> FlattenBranches(this IEnumerable<Policy> policies, IJsonSerializer serializer)
+        {
+            var policiesToFlatten = policies.Where(x => x.Branches != null && x.Branches.Count > 0).ToList();
```
Comment (pascalberger) [inline src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs]: Use any?

```suggestion
            var policiesToFlatten = policies.Where(x => x.Branches != null && x.Branches.Any()).ToList();
```

File: src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs [hunk 2]
```diff
@@ -80,6 +80,38 @@ public static bool DoesSubTypeMatch(this PolicyConfiguration configuration, Poli
...
+            var newPolicies = new List<Policy>();
+
+            foreach (var policy in policies)
+            {
+                if (policy.Branches != null && policy.Branches.Count > 0)
```
Comment (pascalberger) [inline src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs]: Use any?

```suggestion
                if (policy.Branches != null && policy.Branches.Any())
```

File: src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs (hunk 1 above)
Comment (Alea81) [reply src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs]: It's a list, any would read into it count is already there, therefore count > 0 better imho.

File: src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs (hunk 2 above)
Comment (Alea81) [reply src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs]: It's a list, any would read into it count is already there, therefore count > 0 better imho.

**Pull request merged**